docs/extensibility/resources.md

```diff
@@ -0,0 +1,226 @@
+# Resource-based extensions
+
+## Introduction
+
+Resource-based extensions are a means of introducing custom resource pages to busola. To achieve that a Config Map needs to be created in the `kube-public` namespace with a label `busola.io/extension=resource`.
+
+The Config Map needs to have several fields created that define how the resources is handled.
+
+All sections can be provided as either json or yaml.
+
+## Config Map sections
+
+### `resource` section
+
+The resource section is required and contains basic information about the resource - namely kind and api details.
+
+#### Example
+
+```json
+{
+  "kind": "MyResources",
```

Comment:
In my opinion, there should be something different than `kind`. Kind property is always in singular form but there we have to provide kind in plural which can be confusing

Reply:
I've already pointed it [in the comment](https://github.com/kyma-project/busola/pull/1336#discussion_r887997893), it will be fixed in #1395